edfio/edf_signal.py

```diff
@@ -296,6 +314,22 @@ def sampling_frequency(self) -> float:
         """The sampling frequency in Hz."""
         return self._sampling_frequency
 
+    @property
+    def digital_data(self) -> npt.NDArray[np.int16]:
+        """
+        Numpy array containing the digital (uncalibrated) signal values as 16-bit integers.
+
+        To simplify avoiding inconsistencies between signal data and header fields,
+        individual values in the returned array can not be modified. Use
+        :meth:`EdfSignal.update_data` to update raw signal values with calibrated data.
```

Comment:
This is not applicable for `digital_data`, just for `data`, right?

Reply:
This was intended, but I forgot to set the flag to make the array read-only.

Reply:
I changed the documentation, and kept the data writeable. Having it read-only caused problems in the current logic regarding EDF+ annotations.